config/webpack.common.config.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+require('es6-promise').polyfill(); // needed for compatibility with older node versions
+
+var path = require('path');
```

Comment:
I think that's extreme. I think that it would be better to lint the file and disable specific rules that aren't relevant.